docs/markdown/Reference-manual.md

```diff
@@ -2408,6 +2408,9 @@ page](Configuration.md) It has three methods:
 
 - `has(varname)`: returns `true` if the specified variable is set
 
+- `keys()`*(since 0.57.0)*: returns all keys of the configuration data object
+  as an iteratable list.
```

Comment:
I would say "array" since it's not only iteratable but also indexable. Same in the release notes.

Reply:
after you mentioned this idea, I directly found a usecase for it. Changed it accordingly and updated all necessary stuff. Thanks for the hint :)